integrations/tensorflow/bindings/python/pyiree/tf/support/tf_utils_test.py

```diff
@@ -47,36 +47,6 @@ def get_count(self):
 
 class UtilsTests(tf.test.TestCase, parameterized.TestCase):
 
-  @parameterized.named_parameters([
-      {
-          'testcase_name': 'single_backend',
-          'backend_infos': [tf_utils.BackendInfo('iree_vmla')],
-      },
-      {
-          'testcase_name':
-              'multiple_backends',
-          'backend_infos': [
-              tf_utils.BackendInfo('iree_vmla'),
-              tf_utils.BackendInfo('iree_llvmjit')
-          ],
-      },
-  ])
-  def test_artifact_saving(self, backend_infos):
-    with tempfile.TemporaryDirectory() as artifacts_dir:
-      tf_module = ConstantModule()
-      iree_compiled_module = tf_utils.compile_tf_module(
-          tf_module, backend_infos=backend_infos, artifacts_dir=artifacts_dir)
-
-      artifacts_to_check = [
```

Comment:
These file names are not an implementation detail. We are planning on depending on them in benchmarking CI setups and stuff presumably.

(deleting this test for now might be fine though)

Reply:
I added it back after refactoring `_specify_backends` to be accessible outside of `compile_tf_module`.